src/app/join-us/page.tsx

```diff
@@ -1,32 +1,151 @@
+          Visions
+        </Typography>
+        <Typography variant="body2" paragraph>
+          Quis neque, eu et ipsum amet, vel et. Varius integer enim pellentesque
+          ornare pharetra faucibs arcu. Mauris blandit egestas nibh. eu et ipsum
+          amet, vel et. Varius integer enim pellentesque ornare pharetra faucibs
+          arcu. Mauris blandit egestas nibh.
+        </Typography>
+      </Box>
+      <Box className="box-size-40__center">
+        <Box className="display-flex__space-between">
+          <Link href="#" className="link-no-decorated" variant="h5">
+            Volunteers
+          </Link>
+          <Link href="#" className="link-no-decorated" variant="h5">
+            Be A Partner
+          </Link>
+        </Box>
+        <Divider className="divider" />
+      </Box>
+      <Box className="sub-container-2-col">
+        <Box className="box-size-50">
+          <Typography className="sub-header" variant="h5">
+            <strong>
+              Ready to contribute to British Columbia through Open Source
+              projects?
+            </strong>
+          </Typography>
+          <Typography variant="body2" paragraph>
+            Join us at OpenBC and become a part of our active and welcoming
+            community.
+          </Typography>
+        </Box>
+        <Box className="box-size-50">
+          <Box className="display-flex">
+            <Paper className="square">
+              <Box className="box-number">
+                <Typography>1</Typography>
+              </Box>
+            </Paper>
+            <Typography className="margin-left-1" variant="h5">
+              Tell Us About Yourself
+            </Typography>
+          </Box>
+          <Typography className="margin-left-4" variant="body2" paragraph>
+            Fill out our <u>2 minute form</u> so we can help match you with
+            opportunities
+          </Typography>
+          <Box className="display-flex">
+            <Paper className="square">
+              <Box className="box-number">
+                <Typography>2</Typography>
+              </Box>
+            </Paper>
+            <Typography className="margin-left-1" variant="h5">
+              Onboarding
+            </Typography>
+          </Box>
+          <Typography className="margin-left-4" variant="body2" paragraph>
+            Tortor interdumd pulvinar risus. Cursus in odio aenean.
+          </Typography>
+          <Box className="display-flex">
+            <Paper className="square">
+              <Box className="box-number">
+                <Typography>3</Typography>
+              </Box>
+            </Paper>
+            <Typography className="margin-left-1" variant="h5">
+              Have a question?
+            </Typography>
+          </Box>
+          <Typography className="margin-left-4" variant="body2" paragraph>
+            Find you answers in FAQ or email openbc.ca@gmail.com with any
```

Comment:
```suggestion
            Find you answers in FAQ or email{' '}
            <a href="mailto:&#x6f;&#x70;&#x65;&#x6e;&#x62;&#x63;&#x2e;&#x63;&#x61;&#x40;&#x67;&#x6d;&#x61;&#x69;&#x6c;&#x2e;&#x63;&#x6f;&#x6d;">
              openbc.ca AT gmail.com
            </a>{' '}
            with any
```

Few things I'm suggesting here:

1. Adding a `mailto:` link to make contacting us easier
2. Doing HTML character encoding on the e-mail address to minimize the risk of a scrapper adding our e-mail address to mailing/spam lists
https://stackoverflow.com/a/42369575
3. Replacing the `@` character with `AT` for similar reasons as 2

Seems to work as expected when testing my suggested code locally:

https://github.com/OpenBCca/openbc-web/assets/16601729/babb043d-003f-49ed-bbce-7d91459afe21

Reply:
That's great Corry.